README.md

```diff
@@ -4,22 +4,62 @@
 [![Build Status](https://travis-ci.org/TangleID/TangleID.svg?branch=develop)](https://travis-ci.org/TangleID/TangleID)
 ![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg) [![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lernajs.io/)
 
-## Getting Started
+This is the TangleID JavaScript library, which allows you to do the following:
+- Registering and resolving [identity](https://w3c-ccg.github.io/did-spec/)
```

Comment:
Maybe we should not use the word `identity`, because we do not actually register the "identity" to Tangle.
@jserv, How about using the `identifier`? 🤔

Reply:
`Identity` refers to an individual person, much like the person himself. But `identifier`  is a piece of information that helps someone recognize a distinct person. So in my opinion, `identifier` will be more suitable.

Reply:
I agree with @mushroom0117. We shall always use "identifier".

Reply:
Even in UNIX, `identifier` is widely used for machine-specific information. `Identity` is rarely adopted for communication purpose though.

Reply:
Sounds great, let's replace "Identity" with "identifier". 😃 
And I notice core APIs should rename too, I'll send another PR to fix them.
 - registerIdentity() -> registerIdentifier()
 - resolveIdentity() -> resolveIdentifier()